FRC1710-2019/src/main/java/frc/robot/ClawControl.java

```diff
@@ -10,13 +10,23 @@
 
 package frc.robot;
 
+import java.beans.Encoder;
```

Comment:
I don't believe this is the correct Encoder import you want to be using... In fact is this used at all?
If it isn't it should be removed.